packages/core/src/darwin.ts

```diff
@@ -0,0 +1,96 @@
+import { spawn } from "node:child_process";
```

Comment:
Feels like this would have a better home in the `util` folder given it's less of a core moonlight feature and rather a library-like file - not super important though.